Orm/Xtensive.Orm.Manual/Prefetch/PrefetchTest.cs

```diff
@@ -124,50 +124,49 @@ public async Task MainAsyncTest()
       config.Types.Register(typeof(Person).Assembly, typeof(Person).Namespace);
       var domain = Domain.Build(config);
 
-      using (var session = domain.OpenSession())
+      await using (var session = await domain.OpenSessionAsync())

```

Comment:
```suggestion
      var session = await domain.OpenSessionAsync().ConfigureAwait(false);
      await using (session.ConfigureAwait(false))
```

Reply:
There is o need to do this in test code in case we don't synchronously wait for async method completion.